### PR TITLE
Filter featured charms that does not exist

### PR DESCRIPTION
**Pull request by codeEmpress1**
## Done
- filter interface charms that does not exist (returns 404)
## How to QA
- Go to https://charmhub-io-1600.demos.haus/interfaces and click on any interface from the interfaces table, ensure that no i-frames displays with charms that are not found
## Issue / Card https://warthogs.atlassian.net/browse/WD-4144
Fixes #

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ canonicalwebteam.discourse==5.4.2
 canonicalwebteam.image-template==1.3.1
 canonicalwebteam.store-api==4.1.0
 canonicalwebteam.docstring-extractor==1.2.0
-canonicalwebteam.store-base==0.0.2
+canonicalwebteam.store-base==0.0.1
```
Comment (Lukewh) [inline requirements.txt:8]: is this right? Downgrading from 0.0.2 -> 0.0.1

Comment (codeEmpress1) [reply requirements.txt:8]: Yes, it was upgraded automatically, changes in 0.0.2 are breaking and I only want to upgrade when we merge the branch for making beta-app the main app into the main branch.

**Review by Lukewh — APPROVED** [review]
:+1: LGTM

**Comment by Lukewh** [comment]
Still getting errors when trying to load an interface 😬

**Comment by Lukewh** [comment]
No errors now, but I also don't see **any** featured charms

**Comment by codeEmpress1** [comment]
> No errors now, but I also don't see **any** featured charms

This should be fixed now

**Review by Lukewh — APPROVED** [review]
:+1: LGTM again :D

**Pull request merged**